modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenConstants.java

```diff
@@ -202,6 +202,10 @@
     public static final String ENUM_PROPERTY_NAMING = "enumPropertyNaming";
     public static final String ENUM_PROPERTY_NAMING_DESC = "Naming convention for enum properties: 'camelCase', 'PascalCase', 'snake_case', 'UPPERCASE', and 'original'";
 
+    public static final String SERIALIZATION_LIBRARY = "serializationLibrary";
+    public static final String SERIALIZATION_LIBRARY_DESC = "What serialization library to use: 'moshi' (default), or 'gson'";
```

Comment:
This description is Kotlin specific. Can you move it to Kotlin Codegen?

For instance for java client the choice will be between `gson` and `jackson`